### PR TITLE
{math}[GCCcore/10.2.0] Clp 1.17.6 update toolchain

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — CHANGES_REQUESTED** [review]
All 3 files are very different from previous easyconfigs. It doesn't seem like the changes are improvements, but possible just missing that there existed versions already to base the new versions off? These seems to be missing both build and runtime dependencies, as well as not enabling features that seems like a good idea to keep (in particular in Clp)

**Comment by fizwit** [comment]
Osi, and Coinutils were built by me but I never pushed them.  Replace with versions from EB.

File: easybuild/easyconfigs/c/Clp/Clp-1.17.6-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('CoinUtils', '2.11.4'),
+    ('Osi', '0.108.6'),
+]
```
Comment (Micket) [inline easybuild/easyconfigs/c/Clp/Clp-1.17.6-GCCcore-10.2.0.eb:25]: I would suspect that zlib and bzip2 are still required, they tend to sneak in everywhere. Could you check your build log, or `ldd libClp.so` and confirm if they are not present here?

File: easybuild/easyconfigs/c/Clp/Clp-1.17.6-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# John Dey jfdey@fredhtuch.org fizwit@github
+easyblock = 'ConfigureMake'
+
+name = 'Clp'
+version = '1.17.6'
+
+homepage = 'https://github.com/coin-or/Clp'
+description = """Clp (Coin-or linear programming) is an open-source linear programming solver"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```
Comment (Micket) [inline easybuild/easyconfigs/c/Clp/Clp-1.17.6-GCCcore-10.2.0.eb:11]: Add the comment describing this variant perhaps?
```
# NOTE: this easyconfig for Clp provides a minimal build
```

File: easybuild/easyconfigs/c/Clp/Clp-1.17.6-GCCcore-10.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+sanity_check_paths = {
+    'dirs':  ['include',
+              'lib/pkgconfig',
+              'share'],
+    'files': ['include/coin/ClpSolve.hpp',
+              'bin/%(namelower)s',
+              'share/coin/doc/Clp/README',
+              'lib/libClp.%s' % SHLIB_EXT]
+}
+
```
Comment (Micket) [inline easybuild/easyconfigs/c/Clp/Clp-1.17.6-GCCcore-10.2.0.eb:36]: Isn't the CPATH environment variable still relevant here?

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by fizwit** [comment]
@Micket my version of Clp is out of date now.  Closing

**Pull request closed without merging**